src/main/java/twilightforest/client/renderer/TFSkyRenderer.java

```diff
@@ -34,52 +35,52 @@ public TFSkyRenderer() {
 	@SideOnly(Side.CLIENT)
```

Comment:
do not change this method or anything with [VanillaCopy] on it, the code needs to diff cleanly (or near-cleanly) with vanilla so we can track changes in vanilla properly